### ISSUE TITLE
cargo install not working on MacOS

### ISSUE THREAD

**Issue by playdohface** [hlsxx/tukai#3]
Running `cargo install tukai` results in a compiler error on my machine (see the error below).
Cloning the repo and running via `cargo run` however works fine.
Thought I'd let you know, its a cool tool, if needed I am available to run things on my machine to reproduce the issue/verify a fix.

```
error[E0599]: no function or associated item named `is_capslock_on` found for struct `PlatformApi` in the current scope
   --> /Users/karlenkelmann/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tukai-0.1.3/src/windows/typing_window.rs:378:21
    |
378 |     if PlatformApi::is_capslock_on() {
    |                     ^^^^^^^^^^^^^^ function or associated item not found in `PlatformApi`
    |
   ::: /Users/karlenkelmann/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tukai-0.1.3/src/event_handler.rs:63:1
    |
63  | pub struct PlatformApi;
    | ---------------------- function or associated item `is_capslock_on` not found for this struct
```
    
My system:
```
Darwin MacBookAir 24.1.0 Darwin Kernel Version 24.1.0: Thu Oct 10 21:05:14 PDT 2024; root:xnu-11215.41.3~2/RELEASE_ARM64_T8103 arm64
rustc 1.82.0 (f6e511eec 2024-10-15)
cargo 1.82.0 (8f40fc59f 2024-08-21)
```

**Comment by hlsxx** [comment]
Hello, thank you for your report! I really appreciate it. :)
You can try it now—I’ve published v0.1.4. X11 has been completely removed because it hasn’t been well-tested.

**Comment by playdohface** [comment]
It works now, thanks :)
